components/layout/Header.tsx

```diff
@@ -1,191 +1,212 @@
-            textAlign: 'center',
-          }}
           anchor="right"
         >
           {drawer}
         </Drawer>
-        {!matches && (
-        <Box
-          className="desktop-nav-list"
-          sx={{
-            display: 'flex',
-            justifyContent: 'flex-start',
-            py: 1,
-            px: { md: 3, lg: 4 },
-            maxWidth: '1100px',
-          }}
-        >
-          {navItems
-            .filter(
-              (item) => item.text !== 'Access Calculator' && item.text !== 'Donate',
-            )
-            .map((item) => (
-              <Button
-                key={item.text}
-                href={item.href}
-                aria-label={`${item.text.toLowerCase()}`}
-                variant="text"
-                // color="neutral"
-                size="small"
-                className="nav-list__item"
+        <Box sx={{ width: '100%' }}>
+          {matches && (
+            <IconButton aria-label="open sidebar menu" onClick={handleDrawerToggle} sx={{ display: 'flex', width: '100%' }}>
+              <Menu
                 sx={{
-                  whiteSpace: 'nowrap', marginLeft: { md: 0 }, px: { md: 2, lg: 3 }, py: 1,
+                  color: theme.palette.text.light, marginLeft: 'auto',
                 }}
-              >
-                {item.text}
-              </Button>
-            ))}
+                fontSize="large"
+              />
+            </IconButton>
+          )}
+          {!matches && (
+            <Box
+              className="desktop-nav-list"
+              sx={{
+                display: 'flex',
+                justifyContent: 'flex-start',
+                width: '450px',
+                paddingRight: '32px',
+              }}
+            >
+              {navItems.map((item, index) => (
+                <Box
+                  key={item.text}
+                  onMouseEnter={() => handleMouseEnter(index)}
+                  onMouseLeave={handleMouseLeave}
+                >
+                  <Button
+                    href={item.href}
+                    aria-label={`${item.text.toLowerCase()}`}
+                    size="small"
+                    className="nav-list__item"
+                    sx={{
+                      whiteSpace: 'nowrap',
+                      marginLeft: { md: 0 },
+                      px: { md: 2, lg: 3 },
+                      py: 1,
+                      '&:hover': {
+                        color: theme.palette.text.secondary,
+                        backgroundColor: theme.palette.primary.main,
+                      },
+                      '&:active': {
+                        color: theme.palette.text.light,
+                        backgroundColor: '#002138',
+                      },
+                    }}
+                  >
+                    {item.text}
+                  </Button>
+                  {hoveredIndex === index && (
+                    <Box
```

Comment:
I think this is the drop down menu, and I think we need to add more margin around the drop down buttons.